apps/src/code-studio/pd/workshop_dashboard/components/survey_results/matrix_choice_responses.jsx

```diff
@@ -12,7 +12,8 @@ export default class MatrixChoiceResponses extends React.Component {
     answer: PropTypes.object.isRequired,
```

Comment:
only change in this file (same for single_question_choice_responses) is to pass through facilitators object